c2corg_ui/static/js/search/pagination.js

```diff
@@ -47,6 +47,12 @@ app.PaginationController = function($scope, ngeoLocation) {
 
   /**
    * @type {number}
+   * @private
+   */
+  this.maxResultOffset_ = 9970;
```

Comment:
Sure, be you have to take the result size limit into account too (usually 30). But maybe I should check the computation logic again